application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -221,11 +222,20 @@ private RestAction<Message> useChatGptFallbackMessage(ThreadChannel threadChanne
     }
 
     void writeHelpThreadToDatabase(long authorId, ThreadChannel threadChannel) {
+
+        Instant createdAt = threadChannel.getTimeCreated().toInstant();
+        List<String> tagsList =
+                threadChannel.getAppliedTags().stream().map(ForumTag::getName).toList();
+
+        String tags = String.join(", ", tagsList);
```

Comment:
```suggestion
        String tags = threadChannel.getAppliedTags().stream()
                .map(ForumTag::getName)
                .collect(Collectors.joining(","));
```

Reply:
is this a good reason to hold off a 6 months old PR? same with other two comments

Reply:
> is this a good reason to hold off a 6 months old PR? same with other two comments

Yes, it's a good reason. Give reasons if you disagree with the suggestion.